sklearn/manifold/t_sne.py

```diff
@@ -142,10 +142,12 @@ def _kl_divergence(params, P, degrees_of_freedom, n_samples, n_components,
 
     # Q is a heavy-tailed distribution: Student's t-distribution
     n = pdist(X_embedded, "sqeuclidean")
-    n += 1.
```

Comment:
[KLderiv2.zip](https://github.com/scikit-learn/scikit-learn/files/598319/KLderiv2.zip)

(uploaded in .zip as github wasn't taking my pdf - hope that's ok!)